Dockerfile

```diff
@@ -18,7 +18,7 @@ ENTRYPOINT [ "poetry", "run", "flask", "run", "--host=0.0.0.0"]
 FROM base as production
 COPY . ./
 
-ENTRYPOINT [ "poetry", "run", "gunicorn", "--config", "gunicorn_config.py", "wsgi:app"]
+ENTRYPOINT "poetry run gunicorn --config gunicorn_config.py wsgi:app --bind 0.0.0.0:$PORT"
```

Comment:
I can't see anywhere you provide a default `$PORT` value, which means that while this does work on Heroku, it no longer runs according to the instructions in your Readme. We should either update those, or provide a default value (e.g. we could specify that directly above this line `ENV PORT=5000` and if the user passes in a specific value that would be overridden)

It's also worth noting that this string syntax doesn't work locally - try running your containers. Instead, I'd recommend adding an entrypoint script and invoking that. So, you should put your
command in a separate file, e.g. entrypoint.sh, and then invoke it with
ENTRYPOINT ./entrypoint.sh.
• Note that the file needs execute permissions. You can do this with a RUN
chmod +x ./entrypoint.sh command before the
ENTRYPOINT command.